toolkit/imageconfigs/full.json

```diff
@@ -21,12 +22,8 @@
             "Name": "CBL-Mariner Core",
             "PackageLists": [
                 "packagelists/hyperv-packages.json",
-                "packagelists/core-packages-image.json",
-                "packagelists/selinux.json"
```

Comment:
I thought we were keeping selinux enabled for now.

Reply:
We are keeping by default in Core-FIPs, Mariner full and other images. We will work with SELinux-Policy team and get size to minimal. Once work is done we will enable in Core images by default. AFAIK, Max from Chris's team is also looking into it.

Reply:
Until a new package can be made to include only necessary modules for core, removing this seems ok. SELinux-policy is still present in all other configs that Suresh mentioned so progress has still been made towards SELinux enforcing by default. If this will appease the size requirements our tests impose for now then it seems like the best temporary solution.